templates/learn/get-started.html.hbs

```diff
@@ -137,6 +137,5 @@ fn main() {
            src="/static/images/ferris.gif" />
   </div>
 </section>
-{{> components/tools/install-script }}
```

Comment:
This file was removed in a commit from 2019. It seems that until now, handlebars silently ignored the missing component. I believe the new error was introduced in [this PR](https://github.com/sunng87/handlebars-rust/pull/571) The result was that the server would crash on the get-started page and the user would see a 404.

Reply:
I clicked trough a bunch of the pages on the website looking for other occurrances, but couldn't find any. I could've missed something though.